modules/altinnsalary/view/salarylist.php

```diff
@@ -218,6 +223,42 @@
   }
 
   print_work_relation_table($report, $active_employees, true);
+
+  // all employees employed not in this period
+  $address = $_lib["sess"]->dispatchs;
+  foreach ($_GET as $key => $value) {
+    if($key != "show_inactive") {
+      $address .= "&$key=$value";
+    }
+  }
+  $address .= "&periode=$_periode&action_show_salaries=1";
+
+  if(isset($_GET['show_inactive'])) {
+    print "<a href='". $address ."'>Skjul tidligere ansatte</a><br><br>";
+
+    $query_other_employees = "SELECT ap_merged.* FROM (
```

Comment:
Move to a function such as `$report->queryStringForCurrentlyEmployedEmployees();` is